src/components/MintPreview/MintPreview.tsx

```diff
@@ -54,7 +56,9 @@ const MintPreview = (props: MintPreviewProps) => {
 		statusText?: string,
 	) => {
 		const link = zNAToLink(nft.zNA);
-		const parsedLink = getNetworkZNA(link);
+		const parsedLink = IS_DEFAULT_NETWORK
+			? getNetworkZNA(link)
+			: ROUTES.MARKET + '/' + getNetworkZNA(link);
```

Comment:
Just checking -

I noticed in TransferPreview we're appending `ROUTES.MARKET` to the network ZNA regardless of whether or not `IS_DEFAULT_NETWORK === true`. Should we also be doing that here, or is there an edge case I'm missing?

Reply:
Hmm yeah I just looped back on this one. I did this because ```const link = zNAToLink(nft.zNA);``` being set just above line 59 here, appends '/market/` route in the return. Thought ```zNAToLink``` may have been used elsewhere but its not, so I've removed the '/market/' from that util and made the change here to mirror the same as ```TransferPreview``` variable ```parsedLink```

Testing with no network env set:
<img width="650" alt="Screenshot 2022-07-06 at 12 03 06" src="https://user-images.githubusercontent.com/39112648/177536923-695ce535-0f65-4d29-9f6b-fe48ec0962ca.png">


Testing with wilder network env set:
<img width="650" alt="Screenshot 2022-07-06 at 12 09 42" src="https://user-images.githubusercontent.com/39112648/177537195-67523538-6dba-4b5c-bed8-71343bc163f9.png">

Nice! Thanks for pointing that out @colbr